pkg/kubelet/kuberuntime/security_context_others.go

```diff
@@ -0,0 +1,51 @@
+// +build !windows
```

Comment:
same in `security_context_test.go`?

Reply:
I didn't get what you meant.

Reply:
the file security_context_test.go calls into the method you modified. If you compile different version on windows - running these tests on windows would fail. So I suggest to add the same `build !windows` to the file containing tests

Reply:
Also worth renaming the test file with the same pattern as the product code. With `_other` or `_non_windows` suffix.

Reply:
changed.